src/gui/src/views/users/ConflictsView.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-container>
+    <v-card outlined>
+      <v-card-title>Resolve Conflicts</v-card-title>
+      <v-card-text>
+        <div v-if="conflictsStore.conflicts.length === 0">
+          <p>No conflicts to resolve.</p>
+        </div>
+        <div v-else>
+          <div
+            v-for="conflict in conflictsStore.conflicts"
+            :key="conflict.storyId"
+            class="mb-6"
+          >
+            <h3>Conflict for Story: {{ conflict.storyId }}</h3>
+            <div
+              :id="'mergely-editor-' + conflict.storyId"
+              style="height: 300px"
+            ></div>
+            <v-btn
+              color="primary"
+              class="mt-2"
+              @click="getMergedContentForConflict(conflict.storyId)"
+            >
+              Get Merged Content
+            </v-btn>
+            <v-btn
+              color="success"
+              class="mt-2 ml-2"
+              @click="submitResolution(conflict.storyId)"
+            >
+              Submit Resolution
+            </v-btn>
+            <div v-if="mergedContents[conflict.storyId]">
+              <h4>Merged Content:</h4>
+              <pre>{{ mergedContents[conflict.storyId] }}</pre>
+            </div>
+            <v-divider class="my-4"></v-divider>
+          </div>
+        </div>
+      </v-card-text>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup>
+import { ref, onMounted, nextTick } from 'vue'
+import { useConflictsStore } from '@/stores/ConnectorStore'
+
+// Helper functions to dynamically load external assets
+function loadScript(src) {
+  return new Promise((resolve, reject) => {
+    const s = document.createElement('script')
+    s.src = src
+    s.async = true
+    s.type = 'text/javascript'
+    s.onload = resolve
+    s.onerror = () => reject(new Error(`Failed to load script: ${src}`))
+    document.head.appendChild(s)
+  })
+}
+
+function loadCSS(href) {
+  return new Promise((resolve, reject) => {
+    const l = document.createElement('link')
+    l.rel = 'stylesheet'
+    l.type = 'text/css'
+    l.href = href
+    l.onload = resolve
+    l.onerror = () => reject(new Error(`Failed to load CSS: ${href}`))
+    document.head.appendChild(l)
+  })
+}
+
+const conflictsStore = useConflictsStore()
+const mergedContents = ref({})
+
+async function initMergelyGlobally() {
+  if (!window.jQuery) {
+    await loadScript(
+      'https://cdnjs.cloudflare.com/ajax/libs/mergely/5.0.0/mergely.min.js'
+    )
+  }
+  await loadCSS(
+    'https://cdnjs.cloudflare.com/ajax/libs/mergely/5.0.0/mergely.css'
+  )
+}
```

Comment:
This should be cleaned up, there will never be a jQuery, and assets should be downloaded and served locally. Should be installed via pnpm and added to package.json